### PR TITLE
VPN Server Document Correction

### PR DESCRIPTION
**Pull request by SunithaGudisagarIBM**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: website/docs/r/is_vpn_server_route.html.markdown [hunk 1]
```diff
@@ -32,10 +32,9 @@ Review the argument reference that you can specify for your resource.
   When processing this property, check for and log unknown values. Optionally halt processing and surface the error, or bypass the VPN route on which the unexpected property value was encountered.
   - Constraints: The default value is `deliver`. Allowable values are: translate, deliver, drop
 - `destination` - (Required, String) The destination to use for this VPN route in the VPN server. Must be unique within the VPN server. If an incoming packet does not match any destination, it will be dropped.
-  - Constraints: The value must match regular expression `/^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\/(3[0-2]|[1-2][0-9]|[0-9]))$/`
```
Comment (BShrivastav) [inline website/docs/r/is_vpn_server_route.html.markdown:35]: Do we need to remove this ?

**Review by BShrivastav — COMMENTED** [review]
check suggestion please.

File: website/docs/r/is_vpn_server_client.html.markdown [hunk 2]
```diff
@@ -23,8 +23,8 @@ resource "ibm_is_vpn_server_client" "example" {
 ## Argument Reference
 Review the argument references that you can specify for your resource. 
 
-- `vpn_server` - (Required, Forces new resource, String) The VPN server identifier.
-- `vpn_client` - (Required, Forces new resource, String) The VPN client identifier.
+- `vpn_server` - (Required, String) The VPN server identifier.
+- `vpn_client` - (Required, String) The VPN client identifier.
```
Comment (deepaksibm) [inline website/docs/r/is_vpn_server_client.html.markdown:27]: i see these having forcenew true. are we planning to remove it ?

File: website/docs/r/is_vpn_server_route.html.markdown [hunk 3]
```diff
@@ -32,10 +32,9 @@ Review the argument reference that you can specify for your resource.
...
 - `name` - (Optional, String) The user-defined name for this VPN route. If unspecified, the name will be a hyphenated list of randomly-selected words. Names must be unique within the VPN server the VPN route resides in.
   - Constraints: The maximum length is `63` characters. The minimum length is `1` character. The value must match regular expression `/^([a-z]|[a-z][-a-z0-9]*[a-z0-9])$/`
-- `vpn_server_id` - (Required, Forces new resource, String) The VPN server identifier.
+- `vpn_server` - (Required, String) The VPN server identifier.
```
Comment (deepaksibm) [inline website/docs/r/is_vpn_server_route.html.markdown:37]: is forcenew being removed in code ?

File: website/docs/r/is_vpn_server_client.html.markdown (hunk 2 above)
Comment (SunithaGudisagarIBM) [reply website/docs/r/is_vpn_server_client.html.markdown]: added again

File: website/docs/r/is_vpn_server_route.html.markdown (hunk 3 above)
Comment (SunithaGudisagarIBM) [reply website/docs/r/is_vpn_server_route.html.markdown]: added again

File: website/docs/r/is_vpn_server_route.html.markdown [hunk 4]
```diff
@@ -44,11 +43,8 @@ In addition to all argument references listed, you can access the following attr
 - `vpn_route` - The identifier of the VPNServerRoute.
 - `created_at` - (String) The date and time that the VPN route was created.
 - `href` - (String) The URL for this VPN route.
-  - Constraints: The value must match regular expression `/^http(s)?:\/\/([^\/?#]*)([^?#]*)(\\?([^#]*))?(#(.*))?$/`
 - `lifecycle_state` - (String) The lifecycle state of the VPN route.
-  - Constraints: Allowable values are: deleting, failed, pending, stable, updating, waiting, suspended
 - `resource_type` - (String) The resource type.
-  - Constraints: Allowable values are: vpn_server_route
```
Comment (deepaksibm) [inline website/docs/r/is_vpn_server_route.html.markdown:48]: `id` is missing in the attribute

File: website/docs/d/is_vpn_server_route.html.markdown [hunk 5]
```diff
@@ -23,31 +23,22 @@ data "ibm_is_vpn_server_route" "example" {
...
+- `name` - (Optional, String) The VPN route identifier.
 - `vpn_server` - (Required, String) The VPN server identifier.
 
+  ~> **NOTE:** `identifier` and `name` are mutually exclusive.
+
 ## Attribute Reference
 
 In addition to all argument references listed, you can access the following attribute references after your data source is created.
 
 - `id` - The unique identifier of the VPNServerRoute.
```
Comment (deepaksibm) [inline website/docs/d/is_vpn_server_route.html.markdown:36]: id format is vpnserverID/vpnserverRouteID. we can mention that. same in vpnserver route resource as well

File: website/docs/r/is_vpn_server_route.html.markdown [hunk 6]
```diff
@@ -32,10 +32,9 @@ Review the argument reference that you can specify for your resource.
   When processing this property, check for and log unknown values. Optionally halt processing and surface the error, or bypass the VPN route on which the unexpected property value was encountered.
   - Constraints: The default value is `deliver`. Allowable values are: translate, deliver, drop
 - `destination` - (Required, String) The destination to use for this VPN route in the VPN server. Must be unique within the VPN server. If an incoming packet does not match any destination, it will be dropped.
-  - Constraints: The value must match regular expression `/^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\/(3[0-2]|[1-2][0-9]|[0-9]))$/`
 - `name` - (Optional, String) The user-defined name for this VPN route. If unspecified, the name will be a hyphenated list of randomly-selected words. Names must be unique within the VPN server the VPN route resides in.
   - Constraints: The maximum length is `63` characters. The minimum length is `1` character. The value must match regular expression `/^([a-z]|[a-z][-a-z0-9]*[a-z0-9])$/`
```
Comment (deepaksibm) [inline website/docs/r/is_vpn_server_route.html.markdown:36]: constraints should be removed

File: website/docs/r/is_vpn_server_route.html.markdown [hunk 7]
```diff
@@ -32,10 +32,9 @@ Review the argument reference that you can specify for your resource.
   When processing this property, check for and log unknown values. Optionally halt processing and surface the error, or bypass the VPN route on which the unexpected property value was encountered.
   - Constraints: The default value is `deliver`. Allowable values are: translate, deliver, drop
 - `destination` - (Required, String) The destination to use for this VPN route in the VPN server. Must be unique within the VPN server. If an incoming packet does not match any destination, it will be dropped.
```
Comment (deepaksibm) [inline website/docs/r/is_vpn_server_route.html.markdown:34]: forces new resource

File: website/docs/r/is_vpn_server_route.html.markdown [hunk 8]
```diff
@@ -32,10 +32,9 @@ Review the argument reference that you can specify for your resource.
   When processing this property, check for and log unknown values. Optionally halt processing and surface the error, or bypass the VPN route on which the unexpected property value was encountered.
```
Comment (deepaksibm) [inline website/docs/r/is_vpn_server_route.html.markdown:32]: translate, deliver, drop are not nested schema elements. there are available options

**Review by deepaksibm — APPROVED** [review]

**Pull request closed without merging**